Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -177,14 +179,32 @@ public void run() {
             adHeightInDp = (int) adHeightObject;
         }
 
-        if ((adWidthInDp >= VUNGLE_MREC.getWidth() && adHeightInDp >= VUNGLE_MREC.getHeight())) {
-            adSizeType = VUNGLE_MREC;
-        } else if (adWidthInDp >= BANNER_LEADERBOARD.getWidth() && adHeightInDp >= BANNER_LEADERBOARD.getHeight()) {
-            adSizeType = BANNER_LEADERBOARD;
-        } else if (adWidthInDp >= BANNER.getWidth() && adHeightInDp >= BANNER.getHeight()) {
-            adSizeType = BANNER;
+        String adUnitFormat = serverExtras.get(ADUNIT_FORMAT);
```

Comment:
No other network except for Unity checks for ADUNIT_FORMAT, are we sure about string values?

Reply:
Yes, this a risk if they change string value someday. Like Unity, we only check 
 medium_rectangle ad format, otherwise will fallback to banner scape.